designer/server/src/test/scala/pl/touk/nussknacker/test/processes/WithSimplifiedScenarioActivitySpecAsserts.scala

```diff
@@ -0,0 +1,99 @@
+package pl.touk.nussknacker.test.processes
+
+import io.restassured.RestAssured.`given`
+import io.restassured.module.scala.RestAssuredSupport.AddThenToResponse
+import io.restassured.response.ValidatableResponse
+import org.scalatest.freespec.AnyFreeSpecLike
+import pl.touk.nussknacker.test.NuRestAssureMatchers
+import pl.touk.nussknacker.test.base.it.NuItTest
+import pl.touk.nussknacker.test.config.{WithSimplifiedConfigRestAssuredUsersExtensions, WithSimplifiedDesignerConfig}
+
+trait WithSimplifiedScenarioActivitySpecAsserts
+    extends AnyFreeSpecLike
+    with NuItTest
+    with WithSimplifiedDesignerConfig
+    with WithSimplifiedConfigRestAssuredUsersExtensions
+    with NuRestAssureMatchers {
+
+  implicit class VerifyCommentExists[T <: ValidatableResponse](validatableResponse: T) {
+
+    def verifyCommentExists(scenarioName: String, commentContent: String): ValidatableResponse = {
```

Comment:
DRY - it is also in `WithRichScenarioActivitySpecAsserts`

Reply:
It is hard not to DRY in that case, since we have already divisions: 

- `WithRichConfigRestAssuredUsersExtensions` vs `WithSimplifiedConfigRestAssuredUsersExtensions`
- `WithRichDesignerConfig` vs `WithSimplifiedDesignerConfig`